integration-test/Css.ts

```diff
@@ -41,6 +41,10 @@ class CssBuilder<T extends Properties1> {
   get bl() { return this.add("borderLeftStyle", "solid").add("borderLeftWidth", "1px"); }
   get bn() { return this.add("borderStyle", "none").add("borderWidth", "0"); }
 
+  // borderStyleRules
+  get b__dashed() { return this.add("borderStyle", "dashed"); }
```

Comment:
Yeah, totally get that `b---dashed` is what tachyons does...not a super-huge fan of that on the TypeScript side of things. Brainstorming what we could do instead...

I wonder why they didn't do `bDashed` or `bsDashed`...

Wdyt? Other options?

Reply:
I think I'd vote `bsDashed` unless there is some obvious/existing conflicting rule that it overlaps that I'm missing...

Reply:
+1 for `bsDashed`

Reply:
Neat! I like that better too! Will update this PR to reflect this change!